_includes/code/howto/search.generative.py

```diff
@@ -90,27 +125,64 @@
 
 # SingleGenerativePropertiesPython
 # highlight-start
-prompt = "Convert this quiz question: {question} and answer: {answer} into a trivia tweet."
+prompt = (
+    "Convert this quiz question: {question} and answer: {answer} into a trivia tweet."
+)
 # highlight-end
 
 jeopardy = client.collections.get("JeopardyQuestion")
 response = jeopardy.generate.near_text(
-    query="World history",
-    limit=2,
-    single_prompt=prompt
+    query="World history", limit=2, single_prompt=prompt
 )
 
 # print source properties and generated responses
 for o in response.objects:
-    print(o.properties)
-    print(o.generated)
+    print(f"Properties: {o.properties}")
+    print(f"Single prompt result: {o.generative.text}")
 # END SingleGenerativePropertiesPython
 
 # Test results
 assert response.objects[0].collection == "JeopardyQuestion"
-assert len(response.objects[0].generated) > 0
+assert len(response.objects[0].generative.text) > 0
 # End test
 
+# =====================================================
+# ===== SINGLE GENERATIVE EXAMPLE WITH PARAMETERS =====
+# =====================================================
+
+# SingleGenerativeParametersPython
+# highlight-start
+from weaviate.collections.classes.generative import GenerativeConfig, GenerativeParameters
```

Comment:
Is it possible to import them through `weaviate.classes.xxx`? I think that might be the canonical user-facing path. 

Maybe `weaviate.classes.generate`?